.github/workflows/release.yml

```diff
@@ -8,30 +8,31 @@ on:
         description: Bump version number
         options: [major, minor, patch]
         default: patch
-  pull_request:
-    types: [closed]
+  pull_request: { types: [closed] }
 jobs:
   bump:
     if: github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - run: |
-          git config --global user.name Olivaw[bot]
-          git config --global user.email 64868532+iterative-olivaw@users.noreply.github.com
+      - uses: actions/checkout@v3
+      - uses: iterative/setup-cml@v1
+      - name: Bump version
+        run: |
+          cml ci
           git checkout -b bump/$(npm version ${{ github.event.inputs.bump }})
+      - name: Create PR
+        run: |
           git push --set-upstream origin --follow-tags HEAD
-          gh pr create --title "Bump version to $(git describe --tags)" --body "Approve me 🤖"
-          gh pr merge --squash --auto
+          cml pr create --title "Bump version to $(git describe --tags)" --body "Approve me 🤖" --squash
```

Comment:
Are you sure that using CML to release CML is a good idea, taking into account that forges' API can break and cause a catch-22 situation? 🤔

Reply:
instead of `setup-cml` `npm install -g .`

Reply:
Clever! :+1:

Reply:
> both CML and forges' API can break

exactly, in which case the release will fail noisily.